LinkedIn_sc.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+from bs4 import BeautifulSoup
+from urllib.error import HTTPError,URLError
+import requests
+import pandas as pd
+import json
+import time
+
+
+# In[ ]:
+
+
+import pyodbc
+
+
+# In[ ]:
+
+
+def ifNone(value,List):
+    for i in range(len(List)):
+        try:
+            List[i].append(value[i].text.replace('\n','').strip())
+        except IndexError:
+            List[i].append(None)
+    return value,List
+
+def take_info(soup,Title, Company, Location, Time_Published, EmploymentType, Industry, Level, JobFunc):
+    jobs=soup.find_all('li')
+    for index,job in enumerate(jobs):
+        headers={}
+        name=(job.find('h3',{'class':'base-search-card__title'}))
+        c=job.find('h4',class_='base-search-card__subtitle')
+        l=job.find('span',class_='job-search-card__location')
+        try:
+            link=job.find('a').get('href')#,{'class':'base-card__full-link absolute top-0 right-0 bottom-0 left-0 p-0 z-[2]'}
+        except AttributeError:
+            print(job)
+        r=requests.request('get',link,headers=headers)
```

Comment:
you can use `requests.get()` instead like you in `main` and there's less chance to mistype the string `get` and also looks like you don't need to mention headers when you use `requests.get()` especially in this case where they are empty